README.md

```diff
@@ -1,24 +1,11 @@
 # hyper-content-for
 
-Allows you to store HTML for use (and re-use) later.
```

Comment:
I like 'different part of your' was a good addition. I'm not sure about application. My intention for this directive is to use it in an loaded template and have it change something on the 'page' outside of the ui-view.